src/components/UserProfile.jsx

```diff
@@ -171,6 +187,23 @@ export default function UserProfile({
               userId={userId}
             />
           )}
+          {someoneElse && isUserManager && (
+            <Paper
```

Comment:
For consistency, this can be a `Card` instead.

Reply:
Done